samples/snippets/src/test/java/com/google/cloud/examples/securitycenter/snippets/MuteFindingIT.java

```diff
@@ -0,0 +1,234 @@
+    DeleteMuteRule.deleteMuteRule(PROJECT_ID, MUTE_RULE_UPDATE);
+    stdOut = null;
+    System.setOut(out);
+  }
+
+  public static Source createSource(String organizationId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Source source =
+          Source.newBuilder()
+              .setDisplayName("Custom display name")
+              .setDescription("A source that does X")
+              .build();
+
+      CreateSourceRequest createSourceRequest =
+          CreateSourceRequest.newBuilder()
+              .setParent(String.format("organizations/%s", organizationId))
+              .setSource(source)
+              .build();
+
+      Source response = client.createSource(createSourceRequest);
+      System.out.println("Created source : " + response.getName());
+      return response;
+    }
+  }
+
+  public static Finding createFinding(String sourceName, String findingId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Instant eventTime = Instant.now();
+
+      // The resource this finding applies to.  The Cloud Security Command Center UI can link
+      // the findings for a resource to the corresponding Asset of a resource
+      // if there are matches.
+      // TODO(Developer): Replace the below sample resource name
+      String resourceName = "//cloudresourcemanager.googleapis.com/organizations/11232";
+
+      // Start setting up a request to create a finding in a source.
+      Finding finding =
+          Finding.newBuilder()
+              .setParent(sourceName)
+              .setState(State.ACTIVE)
+              .setSeverity(Severity.LOW)
+              .setMute(Mute.UNMUTED)
+              .setFindingClass(FindingClass.OBSERVATION)
+              .setResourceName(resourceName)
+              .setEventTime(
+                  Timestamp.newBuilder()
+                      .setSeconds(eventTime.getEpochSecond())
+                      .setNanos(eventTime.getNano()))
+              .setCategory("LOW_RISK_ONE")
+              .build();
+
+      Finding response = client.createFinding(sourceName, findingId, finding);
+
+      System.out.println("Created Finding: " + response);
+      return response;
+    }
+  }
+
+  public static ListFindingsPagedResponse getAllFindings(String sourceName) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      ListFindingsRequest request = ListFindingsRequest.newBuilder().setParent(sourceName).build();
+
+      return client.listFindings(request);
+    }
+  }
+
+  private static String getOrganizationId() {
+    return "1081635000895";
+  }
+
+  private static String getProject() {
+    return "project-a-id";
+  }
+
+  @Before
+  public void beforeEach() {
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+  }
+
+  @After
+  public void afterEach() {
+    stdOut = null;
+    System.setOut(null);
+  }
+
```

Comment:
Where are you cleaning up all the test resources you created for these samples? Usually we do it in `tearDown` like [this](https://github.com/googleapis/java-bigquery/blob/c37f800941f70d49e3316bd5f4299f3fce654f64/samples/snippets/src/test/java/com/example/bigquery/CreateDatasetIT.java#L63).

Reply:
Please find them here: https://github.com/googleapis/java-securitycenter/pull/719/files#diff-f54e46cf1d9d8390ef2a0bc3feeaacebf6d5a20b63f238405a8f476b7f392523R97-R105
The only deletable resources for these samples are Mute rules.